src/explorer/model/DependenciesMenu.ts

```diff
@@ -45,4 +46,17 @@ export class DependenciesMenu extends Menu implements ITreeItem {
         };
         return treeItem;
     }
+
+    public refresh(): void {
+        this._savePom();
+        mavenExplorerProvider.refresh(this);
+    }
+
+    private _savePom(): void {
+        const textEditors: vscode.TextEditor[] = vscode.window.visibleTextEditors;
+        const textEditor: vscode.TextEditor | undefined = textEditors.find(editor => editor.document.fileName === this.project.pomPath);
```

Comment:
It would be better to use `uri` for future compatibility (when it's not fs-based). Can you give it a try?

something like `document.uri === vscode.uri.file(pomPath)`